googletest/test/gtest_unittest.cc

```diff
@@ -3803,6 +3804,10 @@ TEST(AssertionTest, ASSERT_NO_THROW) {
   EXPECT_FATAL_FAILURE(ASSERT_NO_THROW(ThrowAnInteger()),
                        "Expected: ThrowAnInteger() doesn't throw an exception."
                        "\n  Actual: it throws.");
+  EXPECT_FATAL_FAILURE(ASSERT_NO_THROW(ThrowAnException("blablubb")),
+                       "Expected: ThrowAnException(\"blablubb\") "
+                       "doesn't throw an exception.\n  "
+                       "Actual: it throws: blablubb");
```

Comment:
```suggestion
                       "Actual: it throws.: blablubb");
```

Reply:
In terms of punctuation this is wrong, isn't it? 😄 Why do you think it is better?